### ISSUE TITLE
bug: sbase-behavegui: sbase-gui: test checkboxes behaviour is faulty

### ISSUE THREAD

**Issue by jilaypandya** [seleniumbase/SeleniumBase#2177]
OS: Windows

Steps:

1. Clone the repo locally
2. run one of the following commands `sbase gui` or `sbase behave-gui`
3. Tests get loaded
4. Try to click the test-checkboxes, the tick disappears on mouse release

Troubleshoot:

1. In sbase behave-gui around line 284
```
            cb = tk.Checkbutton(
                text_area,
                text=(row),
                fg="black",
                anchor="w",
                pady=0,
                borderwidth=0,
                highlightthickness=0,
                variable=ara[count],
            )
```

**Comment by mdmintz** [comment]
Thanks for catching this! Looks like a TKinter bug when using colors on Windows. Also found another issue when `sys.executable` contains spaces. I'm looking into fixes for this. Likely with fewer colors involved if necessary.

**Comment by mdmintz** [comment]
All the GUI Apps need modifications. Fixing this will be part of https://github.com/seleniumbase/SeleniumBase/issues/2179

**Comment by mdmintz** [comment]
This was resolved in `4.20.1` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.20.1
